docs/debian-conf.d/main/00_vexim_listmacrosdefs

```diff
@@ -42,3 +42,9 @@ VEXIM_LOCALPART_SUFFIX = +*
 
 CHECK_RCPT_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-rcpt.conf
 CHECK_DATA_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-content.conf
+CHECK_HELO_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-helo.conf
```

Comment:
Is there any reason to want these files outside the usual hierarchy and include them as proposed in this patch?